src/main/kotlin/com/github/pool_party/pull_party_bot/database/dao/PartyDao.kt

```diff
@@ -9,16 +10,31 @@ import org.joda.time.DateTime
 
 interface PartyDao {
 
-    fun getAll(chatId: Long): List<Party>
+    fun getAll(chatId: Long): List<Alias>
 
-    fun getTopLost(chatId: Long): Party?
+    /**
+     * Returns the party alias that is being used most rarely,
+     * if there is some and it hasn't been used for time set in configuration.
+     */
+    fun getTopLost(chatId: Long): Alias?
 
-    fun getById(partyId: Int): Party?
+    fun getByPartyId(partyId: Int): Party?
 
-    fun getByIdAndName(chatId: Long, partyName: String): String?
+    fun getByChatIdAndName(chatId: Long, partyName: String): String?
 
+    /**
+     * @return true on success, false if partyName is already taken.
+     */
     fun create(chatId: Long, partyName: String, userList: List<String>): Boolean
 
+    /**
+     * @return true on success, false if aliasName is already taken or party with partyName doesn't exist.
+     */
+    fun createAlias(chatId: Long, aliasName: String, partyName: String): Boolean
```

Comment:
Would be nice to be able to distinguish aliasName and partyName errors for the more concrete error messages